pkg/crypto/measure.go

```diff
@@ -19,7 +19,7 @@ const (
 )
 
 // TryMeasureBootConfig measures bootconfig contents
-func TryMeasureBootConfig(name, kernel, initramfs, kernelArgs, deviceTree string) {
+func TryMeasureBootConfig(name, kernel, initramfs, kernelArgs, deviceTree, multiboot, multibootArgs string, modules []string) {
```

Comment:
This function should take a `BootConfig` reference as argument instead

Reply:
I agree, but when import bootconfig to the crypto pkg there is a circle include. I think TryMeasureBootConfig() is redundant. When parsing the grub files and creating the bootconfigs TryMeasureData() ist used. So I reworked the part above to use TryMeasureData() as well.

Maybe it makes the crypto pkg more generic. However refactoring the crypto pkg schould be part of an other PR.